lib/config.yml

```diff
@@ -286,6 +289,9 @@ RSpec/ScatteredSetup:
 RSpec/VerifiedDoubles:
   Enabled: false
 
+RSpec/AnyInstance:
```

Comment:
Sure, it's discouraged, but we still use it and it's still nice when we want to stub an external service in a deep association.

Reply:
The intention with this rule is to encourage dependency injection. In practice, we're most often not stubbing our own code.

Reply:
Right. I also think Railsy code tends to add friction to dependency injection, and it becomes less worthwhile much of the time.